### ISSUE TITLE
bot hangs when choosing a summon while starting a free or story quest

### ISSUE THREAD

**Issue by moourns** [Masuzu/ZooeyBot#626]
The bot works fine in event and treasure event mode but when I tried to pick a story quest or free quest it doesn't do anything once the quest is chosen. No attempts to choose a summon are made and there is nothing in the log that indicates any type of error or attempt to choose a summon.

**Comment by Ohwhat00** [comment]
I think this issue is also related to new chrome update, as mentioned https://github.com/Masuzu/ZooeyBot/issues/625 since I also experienced this until I downgraded the chrome version.

The workaround to this(for me anyway) was to already have the party/set you want to use set as default, then it proceeds to pick the summon.
If you make zooey move from one party group/set to another party group/set then it hangs at the summon selection screen.
